### PR TITLE
[NG] Community -> Member page 

### LINKED ISSUE #1085: [NG] Improvise the Community Members Page on layer5-ng
**Current Behavior**
The current [community members](https://layer5ng.netlify.app/community/members) page looks a bit plain and simple.

**Desired Behavior**
Try out some different ways of representing each member card on the page.
Also, incorporate 4 member cards in a single row as compared to 3 member cards currently.
Incorporate LinkedIn profile links along with github and twitter in member card.

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by vinayaksh42**
Signed-off-by: Vinayak Sharma <vinayaksh42@gmail.com>

**Description**
Added 4 cards in a row instead of 3
Added the option of Linkedin Profile
for an example of how to add a Linkedin profile to a member, Check this commit. I have added my own Linkedin profile
Added a little animation to the card, but I think we need some sort of background minimalist SVG animation as pointed out in this [issue ](https://github.com/layer5io/layer5/issues/980)

[link to the member page
](https://deploy-preview-1087--layer5ng.netlify.app/community/members)
This PR fixes #1085 

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/members/vinayak-sharma/index.mdx [hunk 1]
```diff
@@ -6,4 +6,5 @@ twitter: Vinayak47427793
 github: vinayaksh42
 blurb: Mesh Head
 status: Active
+Linkedin: in/vinayak-sharma-141096193/
```
Comment (leecalcote) [inline src/collections/members/vinayak-sharma/index.mdx:9]: Convenient that people don't need to include the full URL. If we're going to do this, let's take the final step and not require the `in/` section either. What do you think, @vinayaksh42?

**Review by tripathyprateek — APPROVED** [review]
Looks great! Thank you @vinayaksh42 . Just take a look at the suggestion Lee has made

File: src/sections/Members-grid/Maintainers.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ const Maintainer = props =>{
                             twitter
                             status
                             meshmate
+                            Linkedin
```
Comment (leecalcote) [inline src/sections/Members-grid/Maintainers.js:21]: Would you mind if the "L" was lowercased to stay inline with the others, @vinayaksh42?

File: src/sections/Members-grid/AllMembers.js [hunk 3]
```diff
@@ -18,6 +18,7 @@ const AllMembers = props =>{
                             twitter
                             status
                             meshmate
+                            Linkedin
```
Comment (leecalcote) [inline src/sections/Members-grid/AllMembers.js:21]: Lowercase "L", if you would.

File: src/components/Profile-card/index.js [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
 import { ProfileCardWrapper } from "./ProfileCard.style";
-import { FaTwitter, FaGithub } from "react-icons/fa";
+import { FaTwitter, FaGithub, FaLinkedin } from "react-icons/fa";
 import MeshMateIcon from "../../svgs/meshmate-icon.svg";
 import MeshMateIconWhite from "../../svgs/meshmate-icon-white.svg";
 import Image from "../image";
 
 const ProfileCard = (props) =>{
-    const {name, position,  github, twitter, status, image_path, meshmate} = props.frontmatter;
+    const {name, position,  github, twitter, status, image_path, meshmate, Linkedin} = props.frontmatter;
```
Comment (leecalcote) [inline src/components/Profile-card/index.js:10]:
```suggestion
    const {name, position,  github, twitter, status, image_path, meshmate, linkedin} = props.frontmatter;
```

File: src/components/Profile-card/index.js [hunk 5]
```diff
@@ -33,6 +33,13 @@ const ProfileCard = (props) =>{
                             </a>
                         </li>
                     )}
+                    { Linkedin && (
+                        <li>
+                            <a href={`https://Linkedin.com/${Linkedin}`}>
```
Comment (leecalcote) [inline src/components/Profile-card/index.js:38]:
```suggestion
                            <a href={`https://Linkedin.com/in/${linkedin}`}>
```

File: src/sections/Members-grid/InactiveMembers.js [hunk 6]
```diff
@@ -18,6 +18,7 @@ const InactiveMembers = props =>{
                             twitter
                             status
                             meshmate
+                            Linkedin
```
Comment (leecalcote) [inline src/sections/Members-grid/InactiveMembers.js:21]:
```suggestion
                            linkedin
```

**Review by leecalcote — APPROVED** [review]
Thank you, @vinayaksh42 and @tripathyprateek

**Comment by vinayaksh42** [comment]
@leecalcote i think the next step in order to enhance the member page should be the addition of svg background [link](https://github.com/layer5io/layer5/pull/1087)

**Pull request merged**